cmd/cli/main.go

```diff
@@ -0,0 +1,361 @@
+						Name:        "S3SecretRefPasswordKey",
+						Usage:       "Optional ; Set S3SecretRefPasswordKey, key inside secret, under which Restic repo password is stored, via cli or via env: ",
+						EnvVars: []string{
+							"S3SECRETREFPASSWORDKEY",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreName,
+						Required:    false,
+						Name:        "restoreName",
+						Usage:       "Optional ; Set restoreName - metadata.Name field, if empty, k8up will generate name, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORENAME",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RunAsUser,
+						Required:    false,
+						Name:        "runAsUser",
+						DefaultText: "0",
+						Usage:       "Optional ; Set user UID, via cli or via env: ",
+						EnvVars: []string{
+							"RUNASUSER",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreToS3Endpoint,
+						Required:    false,
+						Name:        "restoreToS3Endpoint",
+						Usage:       "Optional ; Set restore endpoint, only when using s3 restore method, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORETOS3ENDPOINT",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreToS3Bucket,
+						Required:    false,
+						Name:        "restoreToS3Bucket",
+						Usage:       "Optional ; Set restore bucket, only when using s3 restore method, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORETOS3BUCKET",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreToS3Secret,
+						Required:    false,
+						Name:        "restoreToS3Secret",
+						Usage:       "Optional ; Set restore Secret, only when using s3 restore method, expecting secret name containing key value pair with 'username' and 'password' keys, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORETOS3SECRET",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreToS3SecretUsernameKey,
+						Required:    false,
+						DefaultText: "username",
+						Name:        "RestoreToS3SecretUsernameKey",
+						Usage:       "Optional ; Set RestoreToS3SecretUsernameKey, key inside secret, under which S3 username is stored, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORETOS3SECRETUSERNAMEKEY",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.RestoreToS3SecretPasswordKey,
+						Required:    false,
+						DefaultText: "password",
+						Name:        "RestoreToS3SecretPasswordKey",
+						Usage:       "Optional ; Set RestoreToS3SecretPasswordKey, key inside secret, under which Restic repo password is stored, via cli or via env: ",
+						EnvVars: []string{
+							"RESTORETOS3SECRETPASSWORDKEY",
+						},
+					},
+					&cli.StringFlag{
+						Destination: &restore.Cfg.Namespace,
+						Required:    false,
+						DefaultText: "default",
+						Name:        "namespace",
```

Comment:
Nit: I'd add a short form `-n` because it's what most k8s users would be familiar with.

See Aliases docs: https://github.com/urfave/cli/blob/main/docs/v2/examples/flags.md#alternate-names